content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,675 @@
+kubectl apply -f https://k8s.io/examples/deployments/deployment-with-configmap-two-containers.yaml
+```
+
+Check the pods for this Deployment to ensure they are ready (matching by
+{{< glossary_tooltip text="selector" term_id="selector" >}}):
+```shell
+kubectl get pods --selector=app.kubernetes.io/name=configmap-two-containers
+```
+You should see an output similar to:
+```
+NAME                                        READY   STATUS    RESTARTS   AGE
+configmap-two-containers-565fb6d4f4-2xhxf   2/2     Running   0          20s
+configmap-two-containers-565fb6d4f4-g5v4j   2/2     Running   0          20s
+configmap-two-containers-565fb6d4f4-mzsmf   2/2     Running   0          20s
+````
+
+Expose the Deployment (the `kubectl` tool creates a
+{{<glossary_tooltip text="Service" term_id="service">}}  for you):
+```shell
+kubectl expose deployment configmap-two-containers --name=configmap-service --port=8080 --target-port=80
+```
+
+Use ```kubectl``` to forward the port:
+```shell
+kubectl port-forward service/configmap-service 8080:8080 & # this stays running in the background
+```
+
+Access the service.
+```shell
+curl http://localhost:8080
+```
+
+You should see an output similar to:
+```
+Fri Jan  5 08:08:22 UTC 2024 My preferred color is red
+```
+
+Edit the ConfigMap:
+```shell
+kubectl edit configmap color
+```
+
+In the editor that appears, change the value of key `color` from `red` to `blue`. Save your changes.
+The kubectl tool updates the ConfigMap accordingly (if you see an error, try again).
+
+Here's an example of how that manifest could look after you edit it:
+
+```yaml
+apiVersion: v1
+data:
+  color: blue
+kind: ConfigMap
+# You can leave the existing metadata as they are.
+# The values you'll see won't exactly match these.
+metadata:
+  creationTimestamp: "2024-01-05T08:12:05Z"
+  name: color
+  namespace: configmap
+  resourceVersion: "1801272"
+  uid: 80d33e4a-cbb4-4bc9-ba8c-544c68e425d6
+```
+
+Loop over the service URL for few seconds. 
+```shell
+# Cancel this when you're happy with it (Ctrl-C)
+while true; do curl --connect-timeout 7.5 http://localhost:8080; sleep 10; done
+```
+
+You should see the output change as follows:
+```
+Fri Jan  5 08:14:00 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:02 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:20 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:22 UTC 2024 My preferred color is red
+Fri Jan  5 08:14:32 UTC 2024 My preferred color is blue
+Fri Jan  5 08:14:43 UTC 2024 My preferred color is blue
+Fri Jan  5 08:15:00 UTC 2024 My preferred color is blue
+```
+
+## Update configuration via a ConfigMap in a Pod possessing a sidecar container {#rollout-configmap-sidecar}
+
+The above scenario can be replicated by using a [Sidecar Container](/docs/concepts/workloads/pods/sidecar-containers/) as a helper container to write the HTML file.  
+As a Sidecar Container is conceptually an Init Container, it is guaranteed to start before the main webserver container.  
```

Comment:
webserver is not an English word.

Reply:
(IMO, we can merge this PR even with the “webserver” word as is. I'm a native English speaker and I write “webserver” without the space fairly often).

Reply:
I will make the change from "webserver" to "web server" in the next commit.